tests/garage/envs/test_gym_env.py

```diff
@@ -139,3 +139,13 @@ def test_done_resets_step_cnt():
         if es.last:
             break
     assert env._step_cnt is None
+
+def test_inconsistent_env_infos():
+    env = GymEnv('MountainCar-v0')
+    env.reset()
+    env._env_info = {'k1': 'v1', 'k2': 'v2'}
+    with pytest.raises(RuntimeError,
+                    match='GymEnv outputs inconsistent env_info keys.'):
+        env.step(env.action_space.sample())
+    # check that order of keys don't matter for equality
+    assert env._env_info.keys() == {'k2': 'v2', 'k1': 'v1'}.keys()
```

Comment:
No new line at the end of the file. You can install the pre-commit check locally. For example, `conda activate garage` `pre-commit uninstall` `pre-commit install -t pre-commit` `pre-commit install -t pre-push` `pre-commit install -t commit-msg`

Reply:
Thanks!